pkg/filesystem/fs.go

```diff
@@ -258,16 +258,17 @@ func (fs *Filesystem) Mount(snapshotID string, labels map[string]string) (err er
 	}()
 
 	fsManager, err := fs.getManager(fsDriver)
-	if err != nil {
+	if err != nil && fsDriver != config.FsDriverBlockdev {
```

Comment:
Why ignore the error if 'fsDriver' equals 'config.FsDriverBlockdev'? It seems that 'getManager' always returns a null 'fsDriver' if the error is not equal to nil.

Reply:
We only register nydusd daemons manager for fuse and fscache driver, in tarfs case we do not need nydusd. And I was wondering if it is appropriate to include tarfs in blockdev driver.